src/modules/api.js

```diff
@@ -0,0 +1,20 @@
+const fetchEpisodes = async () => {
+  const url = 'https://api.tvmaze.com/shows/1/episodes';
+  const response = await fetch(url);
+  const data = await response.json();
+
+  const episodesData = data.map((episode) => ({
+    id: episode.id,
+    name: episode.name,
+    image: episode.image,
+    rating: episode.rating,
+    summary: episode.summary,
+    season: episode.season,
+    airdate: episode.airdate,
+  }));
+
+  console.log(episodesData);
```

Comment:
#### OPTIONAL
I would suggest that you consider taking out the `console.log` method included in the `api.js` file. I think that it would help to follow JavaScript best practices.